src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.targets

```diff
@@ -235,22 +235,25 @@ Copyright (c) .NET Foundation. All rights reserved.
     The SDK expects configurations that use runtime compilation to set $(GenerateRazorHostingAssemblyInfo) to true,
     it will be unset by default.
   -->
+  <PropertyGroup>
+    <_GenerateRazorSourceAssemblyInfoDependsOn>RazorGetAssemblyAttributes;$(_GenerateRazorSourceAssemblyInfoDependsOn)</_GenerateRazorSourceAssemblyInfoDependsOn>
+  </PropertyGroup>
+
   <Target 
     Name="RazorGetAssemblyAttributes"
-    AfterTargets="GetAssemblyAttributes"
     Condition="'$(GenerateRazorHostingAssemblyInfo)'=='true' and '$(RazorDefaultConfiguration)'!=''"
     DependsOnTargets="ResolveRazorConfiguration">
     <ItemGroup>
-      <AssemblyAttribute Include="Microsoft.AspNetCore.Razor.Hosting.RazorLanguageVersionAttribute">
+      <_RazorSourceAssemblyAttribute Include="Microsoft.AspNetCore.Razor.Hosting.RazorLanguageVersionAttribute">
```

Comment:
It didn't seem like a good idea to make generate attributes on an assembly a feature of the Razor Sdk. Making this "private"

Reply:
Sure